core/src/main/java/org/apache/accumulo/core/fate/FateTxId.java

```diff
@@ -59,12 +59,4 @@ public static String formatTid(long tid) {
     return FastFormat.toHexString(PREFIX, tid, SUFFIX);
   }
 
-  public static long parseTidFromUserInput(String s) {
```

Comment:
Yup the whole class can be deleted now. I was gonna delete it after this PR was approved and merged, since I had opened another issue associated with deleting that class. But I can also just delete it in this PR if that would be better.

Reply:
> I was gonna delete it after this PR was approved and merged, since I had opened another issue associated with deleting that class.

That is fine, there is an issue so it will not be forgotten and become dead code.